### PR TITLE
Add support for prefix of job name

### PR DESCRIPTION
**Pull request by greggbjensen**
When jobs are skipped and have sub-workflows, conditionally the `/` ending is either there or not.  For example `Database / SQL` when workflow is executed and `Database` when it is skipped.  Adding `prefix` support was needed for this case.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "description": "Wait for the specified jobs in the same run to complete successfully before proceeding",
   "main": "dist/index.js",
   "scripts": {
-    "build": "rm -rf dist bin && ncc build --source-map --license licenses.txt -o dist src/index.ts",
```
Comment (yogeshlonkar) [inline package.json:8]: Could you revert this? this to make sure `Check dist/` workflow validates correct files

**Review by yogeshlonkar — CHANGES_REQUESTED** [review]
Everything looks good, just minor change

**Comment by yogeshlonkar** [comment]
forgot to @ @greggbjensen

File: package.json (hunk 1 above)
Comment (greggjensencart) [reply package.json]: This is fixed; thanks for catching that.  I had to turn it off temporarily.

**Review by yogeshlonkar — APPROVED** [review]

**Pull request merged**